data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,59 @@ mission "Arachi Cartoonist 2"
 				accept
 			label noship
 			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again Captain."`
+
+
+
+mission "Coalition Folklore: Saryd"
+	minor
+	invisible
+	to offer
+		random < 30
+	source
+		attributes "saryd" "folklore"
```

Comment:
Same situation as the news locations. I'm assuming you want this to offer on a planet with "saryd" *and* "folklore," in which case you need two attribute lines. I imagine you'll want to look at the other folklore missions for this same error.
```suggestion
		attributes "saryd"
		attributes "folklore"
```